Source/WebCore/dom/FullscreenManager.cpp

```diff
@@ -44,7 +44,9 @@
 #include "LocalDOMWindow.h"
 #include "LocalFrame.h"
 #include "Logging.h"
+#if ENABLE(MATHML)
```

Comment:
Conditional header includes should come *after* the other includes, not in the middle of the others.